README.md

```diff
@@ -2,21 +2,24 @@ This plugin uses clang for accurately completing C and C++ code.
 
 ## Installation
 
-To build and install in one step, type:
-$ make install
+- To build and install in one step, type: `$ make install`
 
-To build and install in two steps, type:
+- To build and install in two steps, type:
+
+```
 $ make
 $ vim clang_complete.vmb -c 'so %' -c 'q'
+```
 
-Alternatively, you can also put the files in ~/.vim/
+- Alternatively, you can also put the files in `~/.vim/`
 
 You need Vim 7.3 or higher, compiled with python support and ideally, with
 the conceal feature.
 
 ## Minimum Configuration
 
-- Set the `clang_library_path` variable, example
+- Set the `clang_library_path`, which is the directory containing file named
+  libclang.[dll/so/dylib] or the clang shared library file itself, example:
```

Comment:
Actually, for me it was confusing to set-up clang, this is from file:

https://github.com/Rip-Rip/clang_complete/blob/master/doc/clang_complete.txt

lines 240-245

```
       				       	*clang_complete-library_path*
       				       	*g:clang_library_path*
If libclang is not in your library search path, set this to the absolute path
where libclang is available. This should either be a directory containing a
file named libclang.[dll/so/dylib] or the clang shared library file itself.
Default: ""
```

So I used the `find` command, to search for all the extensions and only found the .so file on my linux machine.